### PR TITLE
Moving sites swagger files from RPSaaSMaster private repo to public repo 

### PR DESCRIPTION
**Pull request by bsomeshekar**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Comment by patelkunal** [comment]
azp run

**Comment by bsomeshekar** [comment]
azp run

**Review by rkmanda — APPROVED** [review]
:shipit:

File: specification/edge/resource-manager/Microsoft.Edge/sites/readme.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+## Suppression
+
+``` yaml
+directive:
+  - suppress: MISSING_APIS_IN_DEFAULT_TAG
```
Comment (rkmanda) [inline specification/edge/resource-manager/Microsoft.Edge/sites/readme.md:24]: > MISSING_APIS_IN_DEFAULT_TAG [](http://example.com/codeflow?start=13&length=28)

Specifying this is required for some important automation to run whichis required for signoff.

Comment (bsomeshekar) [reply specification/edge/resource-manager/Microsoft.Edge/sites/readme.md]: Removed MISSING_APIS_IN_DEFAULT_TAG suppression from readme

**Comment by raosuhas** [comment]
Is there any difference from the code checked into the private repo vs what is being checked in here  that needs to be reviewed ?

**Comment by bsomeshekar** [comment]
> Is there any difference from the code checked into the private repo vs what is being checked in here that needs to be reviewed ?

No change, we have copied same files from private repo

File: specification/edge/resource-manager/Microsoft.Edge/sites/readme.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python-track2
```
Comment (msyyc) [inline specification/edge/resource-manager/Microsoft.Edge/sites/readme.md:60]:
```suggestion
  - repo: azure-sdk-for-python
```
Please update it for all the `readme.md` in this PR.

Comment (bsomeshekar) [reply specification/edge/resource-manager/Microsoft.Edge/sites/readme.md]: Updated

**Comment by ramoka178** [comment]
Going by the previous reviewers- Signed Off

**Pull request merged**